code/src/App.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from 'react';
+
+import Header from './components/Header/Header';
+import Hero from './components/Header/Hero';
+import Why from './components/Why';
+import Classes from './components/Classes/Classes';
+import Signup from './components/Signup';
+import Footer from './components/Footer';
+
+import './index.css';
+
+const App = () => {
+  const [screenSize, setScreenSize] = useState(window.innerWidth);
+
+  useEffect(() => {
+    const handleResize = () => setScreenSize(window.innerWidth);
+    window.addEventListener('resize', handleResize);
+    return () => window.removeEventListener('resize', handleResize);
+  }, []);
```

Comment:
Impressive use of useEffect and coming up with this solution for screen size. Really impressed!